tests/preprocess/test_esa_cci.py

```diff
@@ -105,33 +110,28 @@ def test_preprocess(self, tmp_path, cleanup):
         regrid_path = tmp_path / 'regridder.nc'
         regrid_dataset.to_netcdf(regrid_path)
 
-        # build a dummy remap_dict
-        unique_vals = np.unique(dataset.lccs_class.values)
-        remap_dict = dict(zip(unique_vals, np.arange(0, len(unique_vals) * 10, 10)))
-
         processor = ESACCIPreprocessor(tmp_path)
         processor.preprocess(subset_str='kenya', regrid=regrid_path,
-                             parallel_processes=1, cleanup=cleanup,
-                             remap_dict=remap_dict)
+                             cleanup=cleanup)
 
         expected_out_path = (
-            tmp_path / 'interim/esa_cci_landcover_interim'
+            tmp_path / 'interim/static/esa_cci_landcover_interim'
             '/1992_1992-v2.0.7b_testy_test_kenya.nc'
         )
         if not cleanup:
             assert expected_out_path.exists(), \
                 f'Expected processed file to be saved to {expected_out_path}'
 
         expected_out_processed = (
-            tmp_path / 'interim' / 'esa_cci_landcover_'
-            'preprocessed' / 'esa_cci_landcover_kenya.nc'
+            tmp_path / 'interim/static/esa_cci_landcover_'
+            'preprocessed' / 'esa_cci_landcover_kenya_one_hot.nc'
         )
         assert expected_out_processed.exists(), \
             'expected a processed folder'
 
         # check the subsetting happened correctly
         out_data = xr.open_dataset(expected_out_processed)
-        expected_dims = ['lat', 'lon', 'time']
+        expected_dims = ['lat', 'lon']
```

Comment:
why no `time` dimension? don't we need this to concatenate all files? even if we are making them `static`?

Reply:
(see below)